code/modules/mob/living/carbon/xenomorph/castes/spitter/abilities_spitter.dm

```diff
@@ -89,3 +94,57 @@
 
 		prev_turf = T
 		sleep(2)
+
+/datum/action/xeno_action/activable/spray_acid/line/on_cooldown_finish()
+	to_chat(owner, "<span class='xenodanger'>Our dermal pouches bloat with fresh acid; we can use acid spray again.</span>")
+	owner.playsound_local(owner, 'sound/voice/alien_drool2.ogg', 25, 0, 1)
+	return ..()
+
+// ***************************************
+// *********** Scatterspit
+// ***************************************
+/datum/action/xeno_action/activable/scatter_spit
+	name = "Scatter Spit"
+	action_icon_state = "scatter_spit"
+	mechanics_text = "Spits a spread of acid projectiles that splatter on the ground."
+	ability_name = "scatter spit"
+	plasma_cost = 150
+	cooldown_timer = 6 SECONDS
+
+
+/datum/action/xeno_action/activable/scatter_spit/can_use_ability(atom/target, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return
+
+/datum/action/xeno_action/activable/scatter_spit/use_ability(atom/target)
+
+	var/mob/living/carbon/xenomorph/X = owner
+
+	X.face_atom(target) //Face target so we don't look stupid
+
+	if(!do_after(X, 0.25 SECONDS, TRUE, target, BUSY_ICON_DANGER))
+		return fail_activate()
```

Comment:
Increase the doafter time to give marines time to respond

Reply:
I think it's probably fair at 0.25 seconds; it definitely shouldn't have a wind up equivalent to the praetorian's acid cone since it doesn't have anywhere near the same impact.

Reply:
I'd rather introduce it underpowered first, do the change